tests/data/good_config.yaml

```diff
@@ -0,0 +1,29 @@
+billing_interval: monthly
+product_code: foo
+query_interval: 3600
+reporting_api_is_cumulative: true
+reporting_interval: 3600
+usage_metrics:
+  managed_node_count:
+    consumption_reporting: volume
+    dimensions:
+    - dimension: tier_1
+      max: 15
+      min: 0
```

Comment:
I am a big fan of alpha ordering. That said most people tend to expect to count up, i.e. min - max, rather than understanding things in a max to min direction, i.e. subtracting. Also a min max order has the visually nice property that the min and max of adjacent dimensions are located in visually close proximity which max it more likely to spot an inconsistency in continuity.

Reply:
This is a test artifact not an example. Thus there shouldn't be any expectation of order.